packages/archive/imodel-select/src/components/IModelSelector.scss

```diff
@@ -2,87 +2,87 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-
-@import "~@bentley/ui-core/lib/ui-core/index";

```

Comment:
Are any of the changes here real changes or just line endings?

Reply:
i think line endings, this pkg shouldve been untouched (besides the rename), let me look into it

Reply:
okay cool, I see that with a bunch of the /bentley/ packages as well. Not sure what happened with the line endings... I hate line endings lol  We may need to change this repo to have the same gitattributes as the itwinjs-core repo but better.